tests/conftest.py

```diff
@@ -568,3 +568,12 @@ def __exit__(self: chdir, *excinfo: object) -> None:
                 os.chdir(self._old_cwd.pop())
 
     return chdir
+
+
+@pytest.fixture(scope="module")
+def data_path(tmp_path_factory: pytest.TempPathFactory) -> dict[str, object]:
+    """Set up a temporary data directory."""
+    tmp_path = tmp_path_factory.mktemp("data")
+    (tmp_path / "slides").mkdir()
+    (tmp_path / "overlays").mkdir()
+    return {"base_path": tmp_path}
```

Comment:
This name and docstring could be a little more detailed. It just says "a data directory". Is this used for temporary data in general throughout the tests or is it just for visualisation? If the latter then this should be explained.